src/utils/daemon.js

```diff
@@ -46,18 +107,6 @@ export default async function createDaemon (opts) {
     })
   }
 
-  let origins = []
-  try {
-    origins = await ipfsd.api.config.get('API.HTTPHeaders.Access-Control-Allow-Origin')
-  } catch (e) {
-    logger.warn(e)
-  }
-
-  if (!origins.includes('webui://-')) origins.push('webui://-')
-  if (!origins.includes('https://webui.ipfs.io')) origins.push('https://webui.ipfs.io')
-
-  await ipfsd.api.config.set('API.HTTPHeaders.Access-Control-Allow-Origin', origins)
-  await ipfsd.api.config.set('API.HTTPHeaders.Access-Control-Allow-Methods', ['PUT', 'GET', 'POST'])
-
+  await configure(ipfsd)
```

Comment:
I'd assume we'd need to update the config before starting the repo? or at least we'd need to restart the deamon if we do update the config.

Reply:
Yes, I'll change that 😄